aries-core/src/js/components/core/Popover/Popover.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import { Box, Button, Drop } from 'grommet';
+import { Close } from 'grommet-icons';
+
+export const Popover = ({
+  a11yTitle,
+  title,
+  children,
+  footer,
+  targetRef,
+  onClose,
+  ...rest
+}) => {
+  return (
+    <Drop
+      round="xxsmall"
+      role="dialog"
+      elevation="medium"
+      stretch={false}
+      target={targetRef?.current}
+      onClose={onClose}
+      {...rest}
+    >
+      <Box gap="small" direction="row">
+        <Box flex gap="small" pad={{ vertical: 'small', left: 'small' }}>
```

Comment:
As a general rule I think it's better to have the padding/margins in the main container rather than splitting it into left/right pieces amongst other containers. I can see why you split it up at least for the awkward vertical alignment of the text. Can something like this work?
```jsx
<Box gap="xsmall" pad={{ vertical: 'small', left: 'small', right: 'xsmall' }} direction="row">
  <Box flex gap="small">
    {title}
    {children}
    {footer}
  </Box>
  <Button ... margin={{ top: -2px }} />
</Box>
```
This includes a recommendation from Kenny just now to have the gap between text and close button be xsmall.

Reply:
Good comment I can try this out and see thanks @MikeKingdom

Reply:
going with this approach